In [1]:
import scipy.stats

We first need to construct a normal distribution with $\mu = 178$ and $\sigma = 7.7$

In [3]:
specified_normal_dist = scipy.stats.norm(loc=178, scale=7.7)

In [4]:
specified_normal_dist.mean(), specified_normal_dist.std() #checks out!

(178.0, 7.7000000000000002)

5'10 is 177.8 cm

6'1 is 185.4cm

In [5]:
(specified_normal_dist.cdf(185.4) - specified_normal_dist.cdf(177.8)) * 100

34.209468294595311

Thus, approx 34% of the US Male population falls between 5'10 and 6'1.